code/Test_definitions/Traffic_Influence.yaml

```diff
@@ -0,0 +1,1030 @@
+    ## Quick Start
+    The usage of the TI API is based on the management of a "TrafficInfluence"
+    resource, an object containing the intent requested invoking the TI API and
+    that is implemented by the platform configuring the Mobile Network for the
+    optimal routing toward the EAS instance.\
+    The "TrafficInfluence" resource can be created (providing the related
+    parameters that specify the desired intent), queried, modified and
+    deleted.\
+    The TI API is asynchronous, a notification is available providing
+    information about the status of the requested resource.
+    For an Application (identified by "appId") many "TrafficInfluence" resources
+    can be created, e.g. to add multiple users, regions or zones.\
+    \
+    Before starting to use the TI API, the developer needs to know about the
+    below specified details:\
+    \
+    **Base-Url:**
+    The RESTful TI API endpoint, for example
+    [**https://tim-api.developer.tim.it/trafficinfluence**](https://tim-api.\
+    developer.tim.it/trafficinfluence)\
+    \
+    **TrafficInfluence:**
+    This object represents the resource that carries the requirements from the
+    user to be implemented. The TI API is invoked for the life cycle management
+    of this resource (CRUD). The resource contains the intents from the TI API
+    Consumer. Managing this resource, the developer can specify in which
+    geographical location the routing must be applied, toward which application,
+    maybe for a specific set of users or for a limited period of time.\
+    \
+    **trafficInfluenceID:**
+    Identifier for the Traffic Influence resource. This parameter is returned
+    by the TI API and must be used to update it (e.g., adding a Device or
+    deleting it). A different Traffic Influence resource must be created for
+    any Device or Zone or Region. All these resources are related to an
+    Application identified by "appId".\
+    \
+    **apiConsumerId:**
+    Unique identifier for the TI API Consumer.\
+    \
+    **region:**
+    The developer can specify in which geographical area he requires the fastest
+    routing toward the nearest application instance. A "region" is a wide
+    geographical area and it contains one or more "zones". A "zone" is where the
+    Edge Cloud Zone is located. Zones and Regions identifiers are defined and
+    provided by the Telco Operator and can also be used or retrieved with other
+    CAMARA APIs (“MEC Exposure & Experience Management API” and “Simple Edge
+    Discovery”). To add more regions the TI API must be invoked (POST) for each
+    region. New "TrafficInfluence" resources are created (with different
+    "trafficInfluenceID"). All the created resources are aggregated by the
+    Application (identified by "appId").\
+    \
+    **zone:**
+    The developer can specify in which geographical area he requires the fastest
+    routing toward the nearest Application instance. A "zone" is a smaller
```

Comment:
we should tell about the usage of zone and region and how an API user can figure out in which scenario he should be using which one. Also, somewhere we will need to then specify if a user can determine the relationship between zone and region say by another query API which may not be part of the TI API. But without knowing when region to be used and when zone is to be used may leave a gap in API consumer mind.